app/src/test/java/org/feature/fox/coffee_counter/di/services/network/ApiServiceTest.kt

```diff
@@ -149,7 +151,7 @@ class ApiServiceTest {
             .setBody(generateStringBody("Updated Item successfully."))
         mockWebServer.enqueue(response)
 
-        val actualResponse = apiService.updateItem(itemID, body)
+        val actualResponse = apiService.updateItem(bearerToken, itemID, body)
```

Comment:
The test cases should also assert if the authorization header was send correctly to mock webserver